### DISCUSSION TITLE
How can you target a specific node in the chatflow config options when embedding?

### DISCUSSION THREAD

**Discussion by automaton82** [FlowiseAI/Flowise#2012] [Q&A] [answered]
If you have a chatflow with 2 identical nodes, say 2 document loaders, and you want to target a property on just **one** of them (not both), how can you specify the property in the chatflow config in the embed options to only target one and not both?

**Comment by HenryHengZJ** [comment, marked as answer]
Maybe you have figured out, you can specify the node id:

```
{
    "overrideConfig": {
        "openAIApiKey": {
            "chatOpenAI_0": "sk-my-openai-1st-key",
            "openAIEmbeddings_0": "sk-my-openai-2nd-key"
        }
    }
}
```

**Reply by automaton82** [reply]
I did not know that, thank you! It would be cool if the documentation talked about this, I don't think I saw it before.
